main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -27,10 +27,31 @@
     "java.lang.Runtime::runFinalization": "Sys, IO",
     "java.lang.Runtime::totalMemory": "Env, NonDet, IO",
     "java.lang.Runtime::version": "Env",
+    "java.lang.System::arrayCopy": "IO",
+    "java.lang.System::clearProperty": "Env, Sys, IO",
     "java.lang.System::console": "IO",
     "java.lang.System::currentTimeMillis": "IO",
+    "java.lang.System::exit": "Sys, IO",
+    "java.lang.System::gc": "IO",
     "java.lang.System::getenv": "Env, IO",
+    "java.lang.System::getLogger": "Sys, IO",
+    "java.lang.System::getProperties": "Env, Sys, IO",
+    "java.lang.System::getProperty": "Env, Sys, IO",
+    "java.lang.System::setSecurityManager": "Env, Sys, IO",
```

Comment:
Not sure this is Env

Reply:
It checks runtime permissions, so I think that's what I meant by Env, but I agree it shouldn't be Env

https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/System.html#setSecurityManager(java.lang.SecurityManager)